### ISSUE TITLE
Find more useful statistics to be shown 

### ISSUE THREAD

**Issue by alecsci** [azerothcore/wow-statistics#4]
This should be an ongoing issue, where contributors or players should suggest what's meaningful to be shown as a chart statistic.

**Comment by Helias** [comment]
Add Faction statistics -> players per faction
add Guild Statistics -> players per guild

**Comment by locus313** [comment]
would maybe be interesting to also have online players by hour.

**Comment by Helias** [comment]
Yes, but that's not as immediate as the previous ones, because it requires that we save somewhere the data in the db.
it's feasible but it's not immedieate

**Comment by Hacaw** [comment]
@Helias , factions statistics should be easier to add with the given API, since no extra data is required to be sent by the server.
